pkg/api/customization/catalog/templateversion.go

```diff
@@ -71,6 +61,43 @@ func (t TemplateVerionFormatterWrapper) TemplateVersionFormatter(apiContext *typ
 		delete(resource.Values, "upgradeVersionLinks")
 		resource.Values["upgradeVersionLinks"] = linkMap
 	}
+
+	templateVersion := &client.CatalogTemplateVersion{}
```

Comment:
can you unmarshall resource.Values to templateVersion? you can save one api call.

Reply:
Unmarshalled!